### PR TITLE
added recipe for dmd

### PR DESCRIPTION
**Pull request by pirovc**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core

### CODE REVIEW

File: recipes/dmd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package:
+  name: dmd
+  version: "2.065.0"
+
+source:
+  fn: dmd.2.065.0.zip
+  url: http://downloads.dlang.org/releases/2014/dmd.2.065.0.zip
```
Comment (bgruening) [inline recipes/dmd/meta.yaml]: Is it possible to compile these tools and not use the precompiled ones?
Is there already a D compiler in conda?

Comment (pirovc) [inline recipes/dmd/meta.yaml]: I believe so, but I never did it and I do not have the time right now to do so. I am adding this specific version of the D so I can satisfy a dependency for another tool. 

I could not find a D compiler in conda.

**Comment by bgruening** [comment]
@johanneskoester what do you think? I feel more this should be in conda-forge.

**Comment by kyleabeauchamp** [comment]
agree

**Comment by johanneskoester** [comment]
Yes. Can you please move this over to conda-forge/staged-recipes? The reason is that since we depend on conda-forge we try to put all general purpose stuff in there and use bioconda only for bioinformatics-related packages.

**Pull request closed without merging**